apps/elixir_ls_debugger/test/debugger_test.exs

```diff
@@ -271,7 +432,7 @@ defmodule ElixirLS.Debugger.ServerTest do
                        "reason" => "breakpoint",
                        "threadId" => thread_id
                      }),
-                     1_000
+                     5_000
```

Comment:
whoa... that's a heck of a timeout.

Reply:
yeah I hit 1000ms timeout on M2 MBP when working on this SMH

Reply:
is this CPU bound? 
Nice laptop, btw. I just got an M2 max, and it is awesome.